### PR TITLE
RAB-1452: Pause/Resume mass edit

### PR DESCRIPTION
**Pull request by marinfazioakeneo**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by ghoussard — COMMENTED** [review]
You have to implement logic in some `initialize` method. Otherwise, mass edit step will resume at the beginning

**Review by mmetayer — APPROVED** [review]

**Review by Beninho — APPROVED** [review]

**Review by ghoussard — APPROVED** [review]

File: src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml [hunk 1]
```diff
@@ -505,5 +509,8 @@ services:
             - '@akeneo_batch.job_repository'
             -
                 - '@pim_connector.step.delete_attributes'
+            - true
```
Comment (ValentinMumble) [inline src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml:512]:
```suggestion
            - false
```
Not sure the delete_attributes job should be stoppable now

**Review by ValentinMumble — APPROVED** [review]

File: src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml (hunk 1 above)
Comment (StevenVAIDIE) [reply src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml:512]: Agree with you ! If we allow stopping this job when a guy will stop the job the attribute will still be blacklisted

Comment (ValentinMumble) [reply src/Akeneo/Pim/Structure/Bundle/Resources/config/jobs.yml:512]: If we stop the job, the clean removed attribute job won't be triggered? Or we need to implement logic to launch the cleaning job when stopping

**Pull request merged**